src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -416,6 +416,60 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         return this;
     }
 
+
+
+
+    public WebElement swipeTo(SwipeElementDirection s, By by, int attempts){
+
+        int i;
+
+        if (isPresentWait(by)){
+            return driver.findElement(by);
+        }
+        else {
+            for (i = 0; i < attempts; i++) {
+
+                swipeCenter(s);
+                if (isPresentWait(by)) {
+                    return driver.findElement(by);
+                }
+
+            }
+
+        }
+        System.err.println("WARN: Element" + by.toString() + "does not exist!");
+        return null;
+
+
+    }
+
+
+    public WebElement swipeTo(SwipeElementDirection s, String id, int attempts){
```

Comment:
This should overload to the above method, therefore the logic for swiping to an element is only in one place.